src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -78,26 +93,26 @@ private static BufferedImage Mat2BufferedImage(Mat matrix) throws Exception {
 		return bi;
 	}
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws Exception {
 		new Main();
 	}
```

Comment:
uhh, why? This seems like a strange thing to do (both making main() a constructor -- wouldn't this crash if you instanciated two of them??? -- and having main() `throw Exception` will make debugging harder down the line.

Was there some problem you solved by doing this?

Reply:
i put it in because for the params selector i thought we would want a seperate runnable jar, then i realised that if its windows we should just start configuring things. I also put `throws Exception` on the main method because there are lots of try catch's in the constructor, if an error is outside of the try catch's it is because of something serius like a faliure to find the camera